LED_array_settings:
  serial_port:      "/dev/ttyACM0"
  baud_rate:        9600  # make sure to check baud rate before communicating w/ LED array